lib/ProductOpener/Display.pm

```diff
@@ -2725,21 +2751,65 @@ sub search_and_display_products($$$$$) {
 				$cursor = $products_collection->aggregate($aggregate_parameters);
 			}
 			else {
-				$log->debug("Executing MongoDB query", { query => $query_ref, sort => $sort_ref, limit => $limit, skip => $skip }) if $log->is_debug();
+				$log->debug("Executing MongoDB query", $mongodb_query_ref) if $log->is_debug();
```

Comment:
I think this might work, but it would probably be better to name the strucuted logging parameter.

Reply:
name the structured logging?

Reply:
```
$log->debug("Executing MongoDB query", { query => $mongodb_query_ref }) if $log->is_debug();
```

That's what I was thinking.

Reply:
Fixed in 0291c241e17fc1b682529b817da55e90414f5642